### PR TITLE
Update chrome-debug to work with -core@v2 updates

### PR DESCRIPTION
**Pull request by digeff**
Updated chrome-debug to work with -core@v2 updates
Most of the changes are moving code around to a different place so we can make the extensibility work with the new extensibility model from -core@v2.
The code from src/argumentsUpdater.ts, src/components/customizedUninitializedCDA.ts, src/features/reportVersionInformation.ts, src/features/showOverlayWhenPaused.ts, src/launcherAndRuner/chromeLauncher.ts, src/launcherAndRuner/chromeRunner.ts is the same code that we had before refactored into components in a way that -core@v2 can consume it.

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -64,5 +64,19 @@
...
+            "name": "Debug gulpfile",
+            "type": "node",
+            "request": "launch",
+            "protocol": "inspector",
+
+            "cwd": "${workspaceFolder}",
+            "program": "${workspaceFolder}/node_modules/gulp/bin/gulp.js",
+            "args": [
+                "build",
+                "-LLLL"
```
Comment (roblourens) [inline .vscode/launch.json]: What's this arg?

File: src/requestProcessor.ts [hunk 2]
```diff
@@ -0,0 +1,3 @@
+export class AWEFAW {
```
Comment (roblourens) [inline src/requestProcessor.ts]: Huh? :)

File: .vscode/launch.json (hunk 1 above)
Comment (digeff) [reply .vscode/launch.json]: Removing (BTW: That makes gulp log in verbose mode :p )

File: src/requestProcessor.ts (hunk 2 above)
Comment (digeff) [reply src/requestProcessor.ts]: Sometimes the wrong file gets added to a commit... You know how those things go...
Deleted...

File: src/components/customizedUninitializedCDA.ts [hunk 3]
```diff
@@ -0,0 +1,32 @@
+import { BaseCDAState, injectable, IInitializeRequestArgs, UninitializedCDA, ITelemetryPropertyCollector, IDebugAdapterState } from 'vscode-chrome-debug-core';
```
Comment (roblourens) [inline src/components/customizedUninitializedCDA.ts]: Copyright should come first

File: src/components/htmlSourceLogic.ts [hunk 4]
```diff
@@ -0,0 +1,42 @@
+import { ISource, ILoadedSourceTreeNode, IScript, SourceScriptRelationship, utilities, ISourcesRetriever } from 'vscode-chrome-debug-core';
```
Comment (roblourens) [inline src/components/htmlSourceLogic.ts]: Missing copyright

File: src/components/customizedUninitializedCDA.ts (hunk 3 above)
Comment (digeff) [reply src/components/customizedUninitializedCDA.ts]: Fixed

File: src/components/htmlSourceLogic.ts (hunk 4 above)
Comment (digeff) [reply src/components/htmlSourceLogic.ts]: Fixed

**Pull request merged**